### PR TITLE
[FRONTEND][DARKNET]LSTM and GRU support

### PR DESCRIPTION
**Pull request by siju-samuel**
All the RNN operators are completed for Darknet

- [x] RNN #https://github.com/dmlc/tvm/pull/1443
- [x] CRNN  #https://github.com/dmlc/tvm/pull/1492
- [x] LSTM
- [x] GRU

Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from others in the community.

### CODE REVIEW

File: nnvm/python/nnvm/frontend/darknet.py [hunk 1]
```diff
@@ -698,6 +703,123 @@ def _handle_darknet_rnn_layers(self, layer_num, sym):
...
+                new_inputs = _as_list([act_o, h_state])
+                h_state = _darknet_get_nnvm_op(op_name_mul)(*new_inputs, **attrs)
+                self._outs = self._outs + [c_state, h_state]
+                sym = h_state
+            self._sym_array[layer_num] = sym
+            processed = True
+
+        elif LAYERTYPE.GRU == layer.type:
+            if layer.steps > 1:
+                raise NotImplementedError("Currenty support only single step GRU")
```
Comment (PariksheetPinjari909) [inline nnvm/python/nnvm/frontend/darknet.py]: pls correct spell.

File: nnvm/python/nnvm/frontend/darknet.py [hunk 2]
```diff
@@ -698,6 +703,123 @@ def _handle_darknet_rnn_layers(self, layer_num, sym):
             self._sym_array[layer_num] = sym
             processed = True
 
+        elif LAYERTYPE.LSTM == layer.type:
+            if layer.steps > 1:
+                raise NotImplementedError("Currenty support only single step GRU")
```
Comment (PariksheetPinjari909) [inline nnvm/python/nnvm/frontend/darknet.py]: pls correct spell

File: nnvm/tests/python/frontend/darknet/test_forward.py [hunk 3]
```diff
@@ -395,4 +431,6 @@ def test_forward_activation_logistic():
     test_forward_elu()
     test_forward_rnn()
     test_forward_crnn()
+    test_forward_lstm()
+    test_forward_gru()
     test_forward_activation_logistic()
```
Comment (PariksheetPinjari909) [inline nnvm/tests/python/frontend/darknet/test_forward.py]: add end line

**Comment by siju-samuel** [comment]
@PariksheetPinjari909 Thanks for the review. I updated the code. Please check again. TIA.

**Comment by tqchen** [comment]
Thanks @siju-samuel @PariksheetPinjari909 , this is now merged

**Pull request merged**